src/ripple/app/transactors/README.md

```diff
@@ -0,0 +1,217 @@
+
+# Transactors #
+
+## Introduction ##
+
+Each separate kind of transaction is handled by it's own Transactor.
+The Transactor base class provides functionality that is common to
+all derived Transactors.  The Transactor base class also gives derived
+classes the ability to replace portions of the default implementation.
+
+
+# Details on Specific Transactors #
+
+## AddWallet ##
+
+## Change ##
+
+## Offers ##
+
+### CreateOffer ###
+
+### CancelOffer ###
+
+## Payment ##
+
+## SetAccount ##
+
+## SetRegularKey ##
+
+## SetSignerList ##
+
+**Associated JIRA task is [RIPD-182](https://ripplelabs.atlassian.net/browse/RIPD-182)**
+
+In order to enhance the flexibility of Ripple and provide support for enhanced
+security of accounts, native support for "multi-signature" or "multi-sign"
+accounts is required.
+
+Transactions on an account which is designated as multi-sign can be authorized
+either by using the master or regular keys (unless those are disabled) or by
+being signed by a certain number (a quorum) of pre-authorized accounts.
+
+Some technical details, including tables indicating some of the Ripple
+commands and ledger entries to be used for implementing multi-signature, are
+currently listed on the [wiki](https://ripple.com/wiki/Multisign) but will
+eventually be migrated into this document as well.
+
+For accounts which are designated as multi-sign, there must be a list which
+specifies which accounts are authorized to sign and the quorum threshold.
+
+ - Each authorized account has a weight. The sum of the weights of the signers is used to determine whether a given set of signatures is sufficient for a quorum.
```

Comment:
This certainly results from my lack of knowledge of markdown.  I couldn't find a way to get the bullets to work if I wrapped these lines.